blog/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: blog
+  labels:
+    app: blog
+spec:
+  serviceName: blog
+  replicas: 1
+  selector:
+    matchLabels:
+      app: blog
+  template:
+    metadata:
+      labels:
+        app: blog
+    spec:
+{% if deploy %}
+      priorityClassName: production
+{% endif %}
+      containers:
+      - name: blog
+        image: ghost:3.0-alpine
+        imagePullPolicy: Always
+        env:
+          - name: NODE_ENV
+            value: production
+          - name: url
+{% if deploy %}
+            value: http://blog.hail.is
+{% else %}
+            value: http://internal.hail.is/{{ default_ns.name }}/blog
+{% endif %}
+        ports:
+        - containerPort: 2368
+        livenessProbe:
+          httpGet:
+{% if deploy %}
+            path: /
+{% else %}
+            path: /{{ default_ns.name }}/blog/
+{% endif %}
+            port: 2368
+          initialDelaySeconds: 5
```

Comment:
timeoutSeconds defaults to 1. that is probably sufficient, but I wanted to mention that it may be useful to set this number to something higher, or just keep an eye on response times.